util/thread_list_test.cc

```diff
@@ -126,9 +126,11 @@ TEST_F(ThreadListTest, SimpleColumnFamilyInfoTest) {
   const int kLowPriorityThreads = 5;
   const int kSimulatedHighPriThreads = kHighPriorityThreads - 1;
   const int kSimulatedLowPriThreads = kLowPriorityThreads / 3;
+  const int kDelayMicros = 1000000;
   env->SetBackgroundThreads(kHighPriorityThreads, Env::HIGH);
   env->SetBackgroundThreads(kLowPriorityThreads, Env::LOW);
-
+  // Wait 1 second so that threads start
+  Env::Default()->SleepForMicroseconds(kDelayMicros);
```

Comment:
Is there any way to assert threads are started after this delay IF it's needed to proceed? Fix-sized sleep almost always concern me on flakiness (i.e, what if threads need more than 1 sec to start in some rare occasion)?

Reply:
Same for the unit test above

Reply:
I did not change the above unit test. The above one is for the origin threadpool. I will think about how to add the sync point.